frog-utils/src/ReactiveRichText.js

```diff
@@ -0,0 +1,118 @@
+// @flow
+import React, { Component } from 'react';
+import { get, isEqual } from 'lodash';
+import ReactQuill from 'react-quill';
+import 'react-quill/dist/quill.snow.css';
+import { type LogT } from 'frog-utils';
+
+type ReactivePropsT = {
+  path: string | string[],
+  dataFn: Object,
+  logger?: LogT => void,
+  readOnly: boolean,
+  toolbarOptions?: Object[]
+};
+
+export class ReactiveRichText extends Component<ReactivePropsT, ReactivePropsT> {
+
+  quillRef: any;
+
+  modules = () => {
+    const toolbarOptions = this.props.toolbarOptions || [
+      ['bold', 'italic', 'underline', 'strike'],
+      ['blockquote', 'code-block'],
+
+      [{ 'header': 1 }, { 'header': 2 }],
+      [{ 'list': 'ordered'}, { 'list': 'bullet' }],
+      [{ 'script': 'sub'}, { 'script': 'super' }],
+      [{ 'indent': '-1'}, { 'indent': '+1' }],
+      [{ 'direction': 'rtl' }],
+
+      [{ 'size': ['small', false, 'large', 'huge'] }],
+      [{ 'header': [1, 2, 3, 4, 5, 6, false] }],
+
+      [{ 'color': [] }, { 'background': [] }],
+      [{ 'font': [] }],
+      [{ 'align': [] }],
+
+      ['clean']
+    ];
+    return {
+      toolbar: this.props.readOnly ? null : toolbarOptions
+    }
+  };
+
+  opListener = (op, source) => {
+    if (source === this.quillRef) return;
```

Comment:
I prefer { } even around single statements, like { return } - makes it easier to read.

Reply:
yeah it does. Will change that.